src/Widget/Link.php

```diff
@@ -80,4 +80,22 @@ public function createHrefAttribute()
     {
         return new Attribute('href', (string) $this->getUrl());
     }
+
+    /**
+     * Open this link in a modal
+     *
+     * @return $this
+     */
+    public function openInModal(): self
+    {
+        $this->getAttributes()
+            ->set(
+                Attributes::create([
+                    'data-icinga-modal'   => true,
+                    'data-no-icinga-ajax' => true
+                ])
```

Comment:
Why the detour with `Attributes::create`? Please set the attributes individually.

Reply:
Well, why not? The `Attributes::set()` function accepts Attributes instance as well as array attributes, why do the attributes have to be set individually here?